### PR TITLE
Conflict branch

### PR DESCRIPTION
**Pull request by Shut-Oleg**
Створюю кофлікт гілок

### CODE REVIEW

**Comment by Shut-Oleg** [comment]
Вирішив

**Comment by Shut-Oleg** [comment]
default

File: css/style.css [hunk 1]
```diff
@@ -34,6 +34,51 @@ nav li {
 }
 
 li {
+<<<<<<< HEAD
```
Comment (spolischook) [inline css/style.css]: It's not resolved

File: css/style.css [hunk 2]
```diff
@@ -77,3 +122,4 @@ footer li {
      margin-right: 10px;
      display: inline-block;
  }
+>>>>>>> about-me
```
Comment (spolischook) [inline css/style.css]: Start over again, and never commit the conflict to the repo. Review your changes before commit

**Comment by spolischook** [comment]
You cant commit this without fix my comment. I'm seriously, it's crazy idea to merge it.

**Pull request merged**